openedx/core/djangoapps/user_authn/views/tests/test_password.py

```diff
@@ -176,6 +177,27 @@ def test_password_change(self):
         response_dict = json.loads(response.content.decode('utf-8'))
         assert response_dict['success']
 
+    def test_password_change_from_support_tools(self):
+        # Request a password change from Support Tools while logged in from a staff/superuser
+        self.client.logout()
+        UserFactory.create(
+            username='edx',
+            email='edx@example.com',
+            password='edx',
+            is_superuser=True,
```

Comment:
use ddt to show staff, superuser and no staff or superuser case.

Reply:
Added ddt with 4 different cases.